admin-dev/themes/new-theme/js/app/utils/serp/index.js

```diff
@@ -131,8 +131,8 @@ class SerpApp {
   }
 
   checkDesc() {
-    let watchedDescription = this.watchedDescription;
-    let defaultDescription = this.defaultDescription;
+    let {watchedDescription} = this;
+    let {defaultDescription} = this;
```

Comment:
Was that autofixed ? `let {watchedDescription, defaultDescription} = this;`

Reply:
yes, was the autofix

Reply:
Damn why he can't manage to do this auto